libmscore/read206.cpp

```diff
@@ -553,6 +553,9 @@ void readTextStyle206(MStyle* style, XmlReader& e)
                   qDebug("unhandled substyle <%s>", qPrintable(name));
                   return;
                   }
+            int idx = int(ss) - int(Tid::USER1);
```

Comment:
I would add `assert()` and `if` statement to make sure that `idx` is in [0,5] range.

Reply:
It shouldn't ever be outside that range, but I added a check to make sure.